LICENSE

```diff
@@ -291,9 +291,9 @@ convey the exclusion of warranty; and each file should have at least
 the "copyright" line and a pointer to where the full notice is found.
 
     <one line to give the program's name and a brief idea of what it does.>
-Copyright (C) <year> <name of author>
+    Copyright (C) <year>  <name of author>
```

Comment:
Sure, but mark the Copyright as starting from 2002. I think copyrights are generally valid for 75 years or something like that, trademarks on the other hand will have a date range.